core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/LibraryDilution.java

```diff
@@ -429,6 +414,10 @@ public String getBoxPosition() {
     return boxPosition == null ? null : boxPosition.getPosition();
   }
 
+  public void setBoxPosition(DilutionBoxPosition boxPosition) {
+    this.boxPosition = boxPosition;
+  }
+
   @Override
   public Date getLastModified() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Update this method so that its implementation is not identical to "getLastUpdated" on line 183. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4144)